ginipaybank/src/main/java/net/gini/pay/bank/capture/digitalinvoice/onboarding/DigitalInvoiceOnboardingFragment.kt

```diff
@@ -18,9 +21,23 @@ import net.gini.pay.bank.databinding.GpbFragmentDigitalInvoiceOnboardingBinding
  */
 
 /**
- * Internal use only.
+ * When you use the Component API the `DigitalInvoiceOnboardingFragment` displays information about the return assistant to the user.
  *
- * @suppress
+ * You should show the `DigitalInvoiceOnboardingFragment` when the
+ * [DigitalInvoiceFragmentListener.showOnboarding()] is called.
+ *
+ * Include the `DigitalInvoiceOnboardingFragment` into your layout by using the [DigitalInvoiceOnboardingFragment.createInstance()] factory method to create
+ * an instance and display it using the [androidx.fragment.app.FragmentManager].
+ *
+ * A [DigitalInvoiceOnboardingFragmentListener] instance must be available before the `DigitalInvoiceOnboardingFragment` is attached to an activity. Failing
+ * to do so will throw an exception. The listener instance can be provided either implicitly by making the hosting Activity implement the
+ * [DigitalInvoiceOnboardingFragmentListener] interface or explicitly by setting the listener using [DigitalInvoiceOnboardingFragment.listener].
+ *
+ * Your Activity is automatically set as the listener in [DigitalInvoiceOnboardingFragment.onCreate()].
+ *
+ * ### Customizing the Digital Invoice Onboarding Screen
+ *
+ * TODO: PPL-14: Customization guide for return assistant - Android
```

Comment:
still relevant TODO?

Reply:
Yes 🙂. Ticket is currently parked and will be resumed in the future.